Composer/packages/server/src/init.ts

```diff
@@ -1,8 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { getPortPromise } from 'portfinder';
+
 import { start } from './server';
 
-start().catch(err => {
+(async () => {
+  const availablePort = await getPortPromise();
+  process.env.FALLBACK_PORT = availablePort.toString();
+  await start();
+})().catch(err => {
```

Comment:
Why does this need to be duplicated in electron-server?

Reply:
The way electron-server's startup script is structured right now requires it to know the port ahead of time before calling `/server/`'s `start()` method, but I just looked and think it can be rearranged to get rid of this duplication.

I'll ping you when I fix it.